Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -156,7 +156,7 @@ bool AVIDump::CreateVideoFile()
   s_codec_context->time_base.num = 1;
   s_codec_context->time_base.den = VideoInterface::GetTargetRefreshRate();
   s_codec_context->gop_size = 12;
-  s_codec_context->pix_fmt = g_Config.bUseFFV1 ? AV_PIX_FMT_BGRA : AV_PIX_FMT_YUV420P;
+  s_codec_context->pix_fmt = g_Config.bUseBGRA ? AV_PIX_FMT_BGRA : AV_PIX_FMT_YUV420P;
```

Comment:
Repeating what I said on IRC, for posterity: You can get the supported pixel formats from the codec, once it's instantiated. BGRA should probably be used when available since it doesn't require a conversion.